.travis.yml

```diff
@@ -1,13 +0,0 @@
-addons:
-  apt:
-    packages:
-      - xz-utils
-
-env: HADOLINT="$HOME/hadolint"
-
-install:
-  - curl -sLo "$HADOLINT" $(curl -s https://api.github.com/repos/hadolint/hadolint/releases/latest?access_token="$GITHUB_TOKEN" | jq -r '.assets | .[] | select(.name=="hadolint-Linux-x86_64") | .browser_download_url') && chmod 700 ${HADOLINT}
-
-script:
-  - .ci/script.sh
-  - git ls-files --exclude='*Dockerfile' --ignored | xargs --max-lines=1 "$HADOLINT"
```

Comment:
Unused isn't quite the right word; more like not updated yet. hadolint and shellcheck are still good things to run on the repo. I'll open an issue as a reminder.